test/Fuzzing/fuzzer_test.swift

```diff
@@ -0,0 +1,13 @@
+// RUN: %target-build-swift -parse-as-library -sanitize=fuzzer %s -o %t
+// RUN: not %t -only_ascii=1 -max_len=3 | %FileCheck %s
+// CHECK: Crash!
```

Comment:
@jrose-apple actually libFuzzer is currently a part of LLVM, and should be always built whenever LLVM is built. so I guess that's not necessary.